test_scripts/LowVoltage/002_Low_Voltage_and_Shut_Down.lua

```diff
@@ -0,0 +1,214 @@
+---------------------------------------------------------------------------------------------------
+--
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('test_scripts/LowVoltage/common')
+local runner = require('user_modules/script_runner')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local hashId = { }
+local grammarId = { }
+
+--[[ Local Functions ]]
+local function configureHMILevels(pNumOfApps)
+  local apps = {
+    [1] = { 1 },
+    [2] = { 2, 1 },
+    [3] = { 2, 3, 1 },
+    [4] = { 2, 3, 1 }
+  }
+  for k, i in pairs(apps[pNumOfApps]) do
+    local function activateApp()
+      local cid = common.getHMIConnection():SendRequest("SDL.ActivateApp", { appID = common.getHMIAppId(i) })
+      common.getHMIConnection():ExpectResponse(cid)
+      :Do(function() common.cprint(35, "Activate App: " .. i) end)
+    end
+    RUN_AFTER(activateApp, 100 * k)
+  end
+  common.getMobileSession(1):ExpectNotification("OnHMIStatus",
+    { hmiLevel = "FULL", systemContext = "MAIN", audioStreamingState = "NOT_AUDIBLE" })
+
+  if pNumOfApps >= 2 then
+    common.getMobileSession(2):ExpectNotification("OnHMIStatus",
+      { hmiLevel = "FULL", systemContext = "MAIN", audioStreamingState = "AUDIBLE" },
+      { hmiLevel = "LIMITED", systemContext = "MAIN", audioStreamingState = "AUDIBLE" })
+    :Times(2)
+  end
+  if pNumOfApps >= 3 then
+    common.getMobileSession(3):ExpectNotification("OnHMIStatus",
+      { hmiLevel = "FULL", systemContext = "MAIN", audioStreamingState = "NOT_AUDIBLE" },
+      { hmiLevel = "BACKGROUND", systemContext = "MAIN", audioStreamingState = "NOT_AUDIBLE" })
+    :Times(2)
+  end
+end
+
+local function addResumptionData(pAppId)
```

Comment:
@dboltovskyi Please move often used functions into common module and reuse them in scripts.
Functions: `configureHMILevels(), addResumptionData(), checkResumptionData(), checkResumptionHMILevel(), checkAppId()`

Reply:
@aderiabin Accepted. Done in acf103c